### DISCUSSION TITLE
Problema con ezviz BC1 Integrazione

### DISCUSSION THREAD

**Discussion by Capirex666** [indomus/forum#1080] [HUB: Home Assistant]
Buongiorno a tutti.
Ho diverse telecamere della ezviz e si sono integrate facilmente in Home assistant.
Ultimamente ho comprato Il modello BC1 con 2 telecamere e una base.
Ma quando mi chiede "Inserisci le credenziali RTSP per la videocamera ezviz G12253127-G11381321 con IP 192.168.50.106" Mi risponde "Nome Host o indirizzo IP non valido". Dove Sbaglio???
Grazie e buona Giornata
Alessio Giovannini

**Comment by KC-inDomus** [comment]
> Buongiorno a tutti. Ho diverse telecamere della ezviz e si sono integrate facilmente in Home assistant. Ultimamente ho comprato Il modello BC1 con 2 telecamere e una base. Ma quando mi chiede "Inserisci le credenziali RTSP per la videocamera ezviz G12253127-G11381321 con IP 192.168.50.106" Mi risponde "Nome Host o indirizzo IP non valido". Dove Sbaglio??? Grazie e buona Giornata Alessio Giovannini

Credo che la BC1 sia una delle poche EZVIZ senza supporto rtsp

**Reply by Capirex666** [reply]
Quindi Non posso integrarle in Home assistant?

**Reply by KC-inDomus** [reply]
temo di no

**Reply by Capirex666** [reply]
Oramai ti stresso, mi consigli 2 telecamere da esterno magari con batteria?
Grazie per la tua gentilezza.

**Reply by gsarcl366** [reply]
A me le integra automaticamente: come le metto in rete in integrazioni compare quella che ho installato. Prova questa strada: in configurazioni, vai in aggiungi integrazione, vai alla voce ezviz ti chiede di inserire le credenziali di accesso della applicazione ezviz ed il gioco è fatto, riavvii ti compaiono le tue telecamere ezviz per ognuna ti chiede di inserire la password che trovi direttamente sulla telecamera. Spero di esserti stato utile, Giorgio
